samples/RazorSlices.Samples.WebApp/Models/Todo.cs

```diff
@@ -35,7 +35,7 @@ public string Title
 
 internal static class Todos
 {
-    public readonly static Todo[] AllTodos =
+    public readonly static List<Todo> AllTodos =
```

Comment:
This isn't thread-safe. You can either change this to use some thread-safe structure (like `ConcurrentDictionary<int, Todo>` or change the sample app to use Sqlite to store the Todos.